apps/main/src/dex/components/PagePool/PoolDetails/PoolStats/PoolParameters.tsx

```diff
@@ -36,20 +47,162 @@ const PoolParameters = ({
+            </>
+          )}
+
+          <ActionInfo
+            label={t`Virtual price`}
+            value={formatNumber(parameters?.virtualPrice, { maximumFractionDigits: 8, defaultValue: '-' })}
+            valueTooltip={t`Measures pool growth; this is not a dollar value`}
+          />
+        </Stack>
 
+        {/* price oracle & price scale */}
+        {poolData && haveWrappedCoins && (parameters.priceOracle || parameters.priceScale) && !pricesApi && (
+          <Stack spacing={Spacing.sm}>
+            <Title>Price Data</Title>
+            {parameters.priceOracle && (
+              <ActionInfo
+                label={t`Price Oracle`}
+                value={parameters.priceOracle.map((p, idx) => (
+                  <strong key={p}>
+                    {poolData.pool.wrappedCoins[idx + 1]}: {formatNumber(p, { maximumFractionDigits: 10 })}
+                  </strong>
+                ))}
+              />
+            )}
+            {parameters.priceScale && (
+              <ActionInfo
+                label={t`Price Scale`}
+                value={parameters.priceScale.map((p, idx) => (
+                  <strong key={p}>
+                    {poolData.pool.wrappedCoins[idx + 1]}: {formatNumber(p, { maximumFractionDigits: 10 })}
+                  </strong>
+                ))}
+              />
+            )}
+          </Stack>
+        )}
+
+        {!pricesApi && (
+          <Stack spacing={Spacing.sm}>
+            <Title>{t`Pool Parameters`}</Title>
+            {gamma && <ActionInfo label={t`Gamma`} value={formatNumber(gamma, { useGrouping: false })} />}
+            {virtualPrice && (
+              <ActionInfo
+                label={t`A`}
+                value={formatNumber(A, { useGrouping: false })}
+                valueTooltip={
+                  <>
+                    {t`Amplification coefficient chosen from fluctuation of prices around 1.`}
+                    {rampADetails && rampADetails?.isFutureATimePassedToday && (
+                      <>
+                        <br />{' '}
+                        {t`Last change occurred between ${formatDisplayDate(dayjs(initial_A_time))} and ${formatDisplayDate(
+                          dayjs(future_A_time),
+                        )}, when A ramped from ${initial_A} to ${future_A}.`}
+                      </>
+                    )}
+                  </>
+                }
+              />
+            )}
+            {rampADetails && !rampADetails.isFutureATimePassedToday && (
+              <>
+                <ActionInfo
+                  label={t`Ramping A`}
+                  valueTooltip={t`Slowly changing up A so that it doesn't negatively change virtual price growth of shares`}
+                  prevValue={formatNumber(initial_A, { useGrouping: false })}
+                  value={formatNumber(future_A, { useGrouping: false })}
+                />
+                <ActionInfo
+                  label=" "
```

Comment:
not sure how to deal with this one, just keeping it as it was for now

Reply:
Could set the type of `label: ReactNode` in the `ActionInfo` component to `label: ReactNode | undefined`. That way the label can be set to be empty, but it the property is still required and setting as empty has to be done so explicitly as it's an edge case.

Reply:
I don't think it's a good practice to have an actionInfo without a title, I prefer to keep this as an edge case for now